pom.xml

```diff
@@ -385,13 +391,10 @@
                <configuration>
                   <archive>
                      <manifestEntries>
-                        <Implementation-URL>${project.url}</Implementation-URL>
```

Comment:
`Implementation-Vendor` and `Specification-Vendor` are currently empty strings. The added value of having this defined is questionable. `Implementation-URL` is deprecated and should define "the URL from which the extension implementation can be downloaded from". See also https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html#JAR_Manifest. I find the meaning of these attributes rather unclear and therefore I'd give as little information as possible.